recipes-backport/devtools/dnf/dnf_git.bb

```diff
@@ -11,6 +11,7 @@ SRC_URI = "git://github.com/rpm-software-management/dnf.git \
            file://0001-Do-not-hardcode-etc-and-systemd-unit-directories.patch \
            file://0001-Corretly-install-tmpfiles.d-configuration.patch \
            file://0001-Revert-proper-check-of-releasever-when-using-install.patch \
+           file://Aero.repo \
```

Comment:
I think intel-aero.repo would be a better name.  Also this is in the wrong place. The .repo files should belong to their own package(s), not to dnf. Example:

```
rpm -qf /etc/yum.repos.d/fedora.repo 
fedora-repos-25-4.noarch
```

If you add it here it will be a nightmare to consider this as a backport and synchronize with upstream.